backend/src/common/bcws-seed.ts

```diff
@@ -97,6 +94,8 @@ export const handler = (
     homeLocation: faker.helpers.arrayElement(locations),
     workLocation: faker.helpers.arrayElement(locations),
     firstName: faker.person.firstName(),
+    division: faker.company.name(),
```

Comment:
I might still suggest keeping this from an array of enums, just to keep it realistic? 

If it's too much of a pain, all good